### ISSUE TITLE
Inquiry on decoupling physics from framerate

### ISSUE THREAD

**Issue by MotoLegacy** [ZenoArrows/The-Simpsons-Hit-and-Run#52]
Hi Zeno, thank you for your work on the ports of these games. I'm sure you know from your time working on them that the 30 FPS target and frametime-dependent physics, the game (specifically vehicle physics) are very unpredictable when they go above this.

I see you have #48 to track a cap that would alleviate this, but I was curious about the work that would need done to normalize the physics so those who would opt to leave the framerate > 30 could keep the gameplay consistent?

Thanks again for your work.

**Comment by ZenoArrows** [comment]
The port is based on the PC release, is it also an issue there when playing at 60FPS?

**Comment by MotoLegacy** [comment]
> The port is based on the PC release, is it also an issue there when playing at 60FPS?

Yes, 60FPS generally problematic (though obviously still playable). You will get infrequent occurrences of vehicle "bouncing" and vehicles like the canyonaro are unpleasant. Going above 60 stability exponentially worsens. PC users nowadays use a mod launcher to cap the framerate ti alleviate this.
